assets/kube-apiserver/config.yaml

```diff
@@ -199,6 +199,8 @@ apiServerArguments:
   - /etc/kubernetes/secret/server.crt
   tls-private-key-file:
   - /etc/kubernetes/secret/server.key
+  strict-transport-security-directives:
```

Comment:
Can you move this up a few lines to maintain the alphabetical sorting of this list? Other than that, this looks good

Reply:
moved.